app/views/signup/start.html.erb

```diff
@@ -15,6 +15,11 @@ signup_roles = {
   other: t('.other')
 }
 
+# hide student role option when feature flag is on.
+if Settings::Db.store.newflow_feature_flag
```

Comment:
this is a lot of conditional logic in a view template.  can this be pulled out into helpers or a decorator that can be unit tested?  is this IF statement tested somethere?